lib/Form/Label/index.js

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-const Label = ({ label, id, subtitle, className, highlight, active }) => {
+const Label = ({
+  label,
+  id,
+  subtitle,
+  className,
+  highlight,
+  active,
+  highlightHover
+}) => {
   const textElementClasses = cx('form-input__text', {
     'form__input-text--highlight': highlight,
+    'highlight-active': highlightHover,
```

Comment:
Is `highlight-active` meant to be a BEM modifier? If a state based class is required then maybe something like `is-highlighted` is something like that would be inline with other state class formats.